### ISSUE TITLE
Color changed listener

### ISSUE THREAD

**Issue by MFlisar** [martin-stone/hsv-alpha-color-picker-android#21]
I think, this would be nice... Sadly, the observable color is private and has no getter nor does the `ColorPickerView` have a function to forward an custom color observable...

**Comment by martin-stone** [comment]
I'd happily consider a pull request.

**Comment by MFlisar** [comment]
Done

**Comment by martin-stone** [comment]
Thanks for the quick work. I'll have a proper look later. One issue: the `removeColorObserver` function doesn't do what it says. In practice maybe we could live without that function anyway. 

Also, what's the rationale for putting IColorObserver into it's own package? It creates a need for explicit imports everywhere for no real benefit that I can see.

**Comment by MFlisar** [comment]
I'll remove the `removeColorObserver`, I personally don't need it. Just wanted it to be there for the case...

And I'll make the listener an internal class again...

I personally prefer code like:

```
.addListener(new IColorChangedListener(){...})
```

Over colde like:

```
.addListener(new BaseClass.IColorChangedListener(){...})
```

It's cleaner IMHO... I personally use internal listeners only if they are private or protected...

**Comment by martin-stone** [comment]
I meant the `com.rarepebble.colorpicker.interfaces` package (as opposed to just putting it in `com.rarepebble.colorpicker`). Putting it in a separate module seems reasonable.

**Comment by MFlisar** [comment]
I see... Misunderstood you.

It's just because I'm used to it... To group and organize my classes...

In a little library like this it's probably not necessary...

**Comment by MFlisar** [comment]
I corrected the code...

**Comment by martin-stone** [comment]
All merged now, and built and published to jcenter as version 1.7.0. 

Thanks!
